tests/KubernetesClient.Tests/assets/ec-client.key

```diff
@@ -0,0 +1,29 @@
+-----BEGIN EC PARAMETERS-----
```

Comment:
I would consider renaming this file e.g. `elliptic-client.key` so that it is easier to know what it's for _from the filename_ without having to look at how it is used or search up the Git commit message.

Reply:
done.